examples/obstacle_sensor.py

```diff
@@ -1,5 +1,5 @@
 import gym
-import urdfenvs.pointRobotUrdf
+import urdfenvs.point_robot_urdf
 from urdfenvs.sensors.obstacleSensor import ObstacleSensor
```

Comment:
It is simply the import that is wrong.
It should be `from urdfenvs.sensors.obstacle_sensor import ObstacleSensor`
Then you don't need the extra file in sensors.

Reply:
the import is now correct, and the ObstacleSensor file is no longer with us. obstacle_sensor is.